vimrc

```diff
@@ -43,6 +43,7 @@ augroup vimrcEx
   " Set syntax highlighting for specific file types
   autocmd BufRead,BufNewFile Appraisals set filetype=ruby
   autocmd BufRead,BufNewFile *.md set filetype=markdown
+  autocmd BufRead,BufNewFile .{jscs,jshint,eslint}rc setfiletype json
```

Comment:
Are there any differences between `setfiletype json` and `set filetype=json`? Should this be consistent with the preceding lines?